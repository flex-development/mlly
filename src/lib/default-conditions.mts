/**
 * @file defaultConditions
 * @module mlly/lib/defaultConditions
 */

import type { Condition } from '@flex-development/pkg-types'

/**
 * Default conditions.
 *
 * @see {@linkcode Condition}
 * @see https://nodejs.org/api/packages.html#conditional-exports
 *
 * @const {Set<Condition>} defaultConditions
 */
const defaultConditions: Set<Condition> = new Set(['node', 'import'])

export default defaultConditions
