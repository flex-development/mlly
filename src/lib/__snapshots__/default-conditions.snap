// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`unit:lib/defaultConditions > should contain default conditions 1`] = `
Set {
  "node",
  "import",
}
`;
